packages/fasd/Makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- ../Makefile
++++ ./Makefile
+@@ -1,4 +1,4 @@
+-PREFIX?= /usr/local
++PREFIX?= /data/data/com.termux/files/usr
```

Comment:
since `PREFIX` is defined [here](https://github.com/termux/termux-packages/blob/master/build-package.sh#L293), is this really needed?